hitbox-tarantool/src/backend.rs

```diff
@@ -0,0 +1,179 @@
+use std::io::{Error, ErrorKind};
+
+use async_trait::async_trait;
+use derive_builder::Builder;
+use hitbox_backend::{
+    serializer::{JsonSerializer, Serializer},
+    BackendError, BackendResult, CacheBackend, CacheableResponse, CachedValue, DeleteStatus,
+};
+use rusty_tarantool::tarantool::{Client, ClientConfig, ExecWithParamaters};
+use serde::{Deserialize, Serialize};
+
+/// Tarantool cache backend based on rusty_tarantool crate.
+///
+/// # Examples
+/// ```
+/// use hitbox_tarantool::TarantoolBackend;
+///
+/// #[tokio::main]
+/// async fn main() {
+///     let mut backend = TarantoolBackendBuilder::default()
+///         .build()
+///         .unwrap();
+///     backend.init().await.unwrap();
+/// }
+/// ```
+#[derive(Clone, Builder)]
+pub struct TarantoolBackend {
+    #[builder(default = "\"hitbox\".to_string()")]
+    user: String,
```

Comment:
Should it be an optional field, or does Tarantool always require a username and password?

Reply:
Tarantool is not required, but the client for Tarantool requires